src/Components/Atbash/AtbashInput.js

```diff
@@ -1,72 +1,126 @@
-        })
-    }
+class Atbash extends Component {
+  state = {
+    inputStr: null,
+    outputStr: null,
+  };
+  handleChange = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    });
+  };
+  handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(this.state);
 
-    backwardsAlpha= () => {
-        var i;
-        let letters = ""
-        for(i = 25; i >= 0; i--) {
-            letters+=String.fromCharCode(i+65);
+    var strOut = "";
+    var str = this.state.inputStr;
+    var atbashCode = 0;
+
+    for (var i = 0; i < str.length; i++) {
+      var c = str[i];
+      if (c.match(/[a-z]/i)) {
+        let code = str.charCodeAt(i);
+        if (code >= 65 && code <= 90) {
+          code = code - 65;
+          atbashCode = ((26 - code + 25) % 26) + 65;
+          c = String.fromCharCode(atbashCode);
+        }
+        // Lowercase letters
+        else if (code >= 97 && code <= 122) {
+          code = code - 97;
+          atbashCode = ((26 - code + 25) % 26) + 97;
+          c = String.fromCharCode(atbashCode);
         }
-        console.log(letters)
-        return letters
-    } 
+      }
+      strOut += c;
+    }
+    console.log("strOut is: " + strOut + "\n");
+    this.setState({
+      outputStr: strOut,
+    });
+  };
 
-    render(){
-        return(
-            <div className="container" id="atbash_cipher">
-            <p className="title">ATBASH CIPHER: </p>
-                <LetterBox default={true} offset={0}></LetterBox>
-                <LetterBox default={false} offset={0} length={17} letters={this.backwardsAlpha()}></LetterBox>
-                <form onSubmit={this.handleSubmit}>
-                    <label htmlFor="input">Input your plain text: </label>
-                    <input type="text" id="inputStr" onChange={this.handleChange} />
-                    <button>Submit</button>
-                </form>
-                <p>Your ciphered Atbash text is: {this.state.outputStr}</p>
-            </div>
-        )
+  backwardsAlpha = () => {
+    var i;
+    let letters = "";
+    for (i = 25; i >= 0; i--) {
+      letters += String.fromCharCode(i + 65);
     }
+    console.log(letters);
+    return letters;
+  };
+
+  render() {
+    return (
+      <div className="container">
+        <div className="vertical-spacing">
+          <p className="subtitle">
+            So in Caesar cipher, each letter in the alphabet turned into the
+            letter x places after it.
+          </p>
+          <LetterBox default={true} length={11} offset={0}></LetterBox>
+          <div classname="row">
+            <Arrow />
+            <Arrow />
+            <Arrow />
+            <Arrow />
+            <Arrow />
+          </div>
+          <LetterBox default={true} offset={0} length={17}></LetterBox>
+          <br />
+          <p className="subtitle">
+            Some Hebrew guy thought he could do the Greeks one better, and
+            flipped his alphabet so they could just do this:
```

Comment:
Though the Atbash cipher's history is difficult to find information on, I think changing the tone might be in order. Maybe clarify that we don't know exactly who created the Cipher. For example:

"We don't know much about the history of the Atbash cipher, but we do know it was created by a group of Hebrew people well before the common era."

Just food for thought. The trouble with this example is that it isn't super kid-friendly, which is a concern.